src/cloudkit.cs

```diff
@@ -19,15 +19,155 @@ interface CKAsset : NSCoding, NSSecureCoding {
 		[Export ("fileURL", ArgumentSemantic.Copy)]
 		NSUrl FileUrl { get; }
 	}
+	
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12, onlyOn64 : true)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface CKUserIdentityLookupInfo : NSSecureCoding, NSCopying
+	{
+		[Static, Internal, Export ("initWithEmailAddress:")]
+		IntPtr _FromEmail (string emailAddress);
+
+		[Static, Internal, Export ("initWithPhoneNumber:")]
+		IntPtr _FromPhoneNumber (string phoneNumber);
+
+		[Static, Internal, Export ("initWithUserRecordID:")]
+		CKUserIdentityLookupInfo FromRecord (CKRecordID userRecordID);
```

Comment:
that one can be switched back to a _normal_ constructor in the bindings, i.e. 

    [Export ("initWithUserRecordID:")]
    IntPtr Constructor (CKRecordID userRecordID)